src/GridFieldOrderableRows.php

```diff
@@ -656,7 +688,13 @@ protected function reorderItems($list, array $values, array $sortedIDs)
                 $record = $currentSortList[$targetRecordID];
                 if ($record->$sortField != $newSortValue) {
                     $record->$sortField = $newSortValue;
-                    $record->write();
+
+                    if ($this->getRepublishLiveRecords() && $record->isLiveVersion()) {
+                        $record->write();
+                        $record->copyVersionToStage(Versioned::DRAFT, Versioned::LIVE, true);
+                    } else {
+                        $record->write();
+                    }
```

Comment:
You could probably put the `$record->write()` before the condition to avoid writing it twice

Reply:
You might have to do `$wasLive = $record->isLiveVersion()` before the write though.

Reply:
It only writes once? I can't write before the check since then the check for `isLiveVersion()` would fail.

Reply:
@ScopeyNZ updated